app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
```

Comment:
Better to use brackets instead of \
```suggestion
        return (f"{{Name: {self.name}, "
               f"Health: {self.health}, "
               f"Hidden: {self.hidden}}}")
```

Reply:
Flake8 have an error 
E128 continuation line under-indented for visual indent